technical-report/sections/00-intro.adoc

```diff
@@ -0,0 +1,54 @@
+
+In this Technical Report the two International Standards will be examined in their universality and not by comparing specific implementations such as Microsoft Office or OpenOffice.org/LibreOffice. For this reason, various examples have been developed using a simple XML editor which supports both standards. The names of specific implementations may be used in the use cases to illustrate the real world scenario behind the use case. The figures in this Technical Report are created for illustration purposes, using available tools such as OpenOffice 3.* and Microsoft Office 2010. It should not be assumed that the current versions of these implementations support all the features needed to implement the use case, especially the document standards and the translation between them.
+
+Several use cases do not mention existing tools, but rather use abstract names such as document format A (DF-A) and document format B (DF-B).
+
+This Technical Report begins with a presentation of typical use cases characterizing scenarios where specific features supported by both document formats are used. It then analyses the most important features of one document format and shows how those features can best be represented in the other format. It then reviews the concepts and various features of the two document formats in order to provide a good understanding of the formats' common features and especially their differences. Most features can be translated to the other format with varying degrees of fidelity. For the most important features, this Technical Report provides detailed information on the implementation of the feature and the extents to which that feature can be translated, including typical translation rules. Finally, an overview summing up the most important results and deriving guidelines for the translation between both formats concludes this Technical Report.
+
+The following abbreviations are used throughout this Technical Report:
+
+* ODF, which stands for OpenDocument Format (<<iso-iec_26300>>);
+* OOXML, which stands for Office Open XML (ISO/IEC 29500:2008).
+
+It is hoped that this Technical Report will be useful in understanding how the ODF and OOXML International Standards compare and how their functionality can be mapped between the two formats. It is a necessary step to the goal of helping achieve interoperability and harmonization between the two formats.
+
+=== History of ODF and OOXML
```

Comment:
Let's keep this subclause with 1 paragraph.

Reply:
Done